sw/device/lib/testing/test_framework/ottf_macros.h

```diff
@@ -9,5 +9,9 @@
 #define OTTF_NV_SCRATCH _non_volatile_scratch_start
 #define OTTF_HALF_WORD_SIZE (OTTF_WORD_SIZE / 2)
 #define OTTF_CONTEXT_SIZE (OTTF_WORD_SIZE * 30)
+#define OTTF_TASK_DELETE_SELF_OR_DIE \
+  ottf_task_delete_self();           \
+  abort();                           \
+  CHECK(false);
```

Comment:
Isn't `CHECK(false)` dead-code? I thought `abort()` is `while(true) { wfi }`

Reply:
yes, fixed in #15249.